numerics/fast_fourier_transform_test.cpp

```diff
@@ -190,6 +198,23 @@ TEST_F(FastFourierTransformTest, Vector) {
               AlmostEquals(4 * π / FFT::size * Radian / Second, 24));
 }
 
+TEST_F(FastFourierTransformTest, Inverse) {
+  constexpr int n = 4;
+  constexpr Time Δt = 1 * Second;
+  std::array<Voltage, n> u{{1 * Volt, 0 * Volt, 1 * Volt, 0 * Volt}};
+  FastFourierTransform<Voltage, Instant, n> const U(u, Δt);
+
+  FastFourierTransform<Voltage, AngularFrequency, n> const nu(
```

Comment:
Use Greek nu?

Reply:
No, this is n times u. Renamed u to v for clarity.